tnb/base_client.py

```diff
@@ -45,27 +45,16 @@ def fetch(self, resource: str, **kwargs: dict) -> Union[dict, list]:
         GET a `resource` from a Node
         Return response as Python object
         """
-        return self.send_request("GET", resource, **kwargs)
-
-    def fetch_multiple_page(self, resource: str, **kwargs) -> list:
-        """
-        GET a complete `resource` from a Node where this resource is splited into
-        multiple pages.
-        Return response as Python object
-        """
-        complete_response = []
-        current_response = self.fetch(resource)
-        complete_response += current_response["results"]
-        next_url = current_response["next"]
+        params = kwargs.get("params", {})
+        page = kwargs.pop("page", 0)
+        limit = kwargs.pop("limit", 0)
+        offset = (page - 1) * limit
```

Comment:
Let user specify `offset` and `limit` would be nicer. Assuming user want to a block 1615, they would choose the `offset = 1615` and `limit = 1` rather than calculating `page`, pass it to here and recalculate the offset.